helm/prometheus-rules/templates/kaas/tenet/alerting-rules/vertical-pod-autoscaler.rules.yml

```diff
@@ -33,15 +33,25 @@ spec:
         description: |-
           {{`Flux HelmRelease {{ $labels.name }} in ns {{ $labels.exported_namespace }} on {{ $labels.installation }}/{{ $labels.cluster_id }} is stuck in Failed state.`}}
         runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/fluxcd-failing-helmrelease/
-      expr: gotk_reconcile_condition{type="Ready", status="False", kind="HelmRelease", cluster_type="management_cluster", exported_namespace!="flux-giantswarm", exported_namespace!~"org-t-.*", name=~".*(vertical-pod-autoscaler-crd)"} > 0
+      {{- $components := "(vertical-pod-autoscaler-crd)" }}
+      expr: |
+        (
+          label_replace(gotk_reconcile_condition{type="Ready", status="False", kind="HelmRelease", cluster_type="management_cluster", exported_namespace!="flux-giantswarm", exported_namespace!~"org-t-.*", name=~"(.+)-{{ $components }}"}, "cluster_id", "$1", "name", "(.+)-{{ $components }}")
+          * on(cluster_id) group_left(provider)
+          sum(
+              label_replace(
+                capi_cluster_info, "provider", "vsphere", "infrastructure_reference_kind", "VSphereCluster"
+              )
+          ) by (cluster_id, provider)
+        ) > 0
       for: 20m
       labels:
         area: kaas
         cancel_if_outside_working_hours: "true"
         cancel_if_kube_state_metrics_down: "true"
         cancel_if_monitoring_agent_down: "true"
         severity: page
-        team: tenet
+        team: {{ include "providerTeam" . }}
```

Comment:
Shouldn't this remain a Tenet alert?

Reply:
No because otherwise the extracted team label for onprem gets overidden to tenet and it pages the cloud rotation even for onprem clusters

Reply:
But that component is owned by @giantswarm/team-tenet isnt it? Regardless of cloud or onprem. The problem here is because tenet isn't big enough to have our own rota for on-call. Are @giantswarm/team-rocket ok with dealing with alerts for our component?

Reply:
I don't know but I know that @T-Kukawka clearly does not want to be paged by those :D

Reply:
this has been an agreement for a while already, that any of those 'shared' alerts will go to provider teams for now until we have separate oncall, the same is with KSM or any other components

Reply:
@JosephSalisbury can confirm 😄